packages/web/src/components/result/ResultList.js

```diff
@@ -12,31 +12,35 @@ class ResultList extends Component {
 
 	state = {
 		hasImage: false,
+		isSmall: false,
 	};
 
 	componentDidMount() {
 		let hasImage = false;
+		let isSmall = false;
 		React.Children.forEach(this.props.children, (o) => {
```

Comment:
Instead of using `forEach` here, you should use methods like `every` or `some` - which don't run till loop completion. It will make the code more readable as then you can remove `!hasImage` from the `if` condition.

Reply:
Updated the PR.
Please take a look now.